docs/_future/manifest.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Manifest
+lang: en
+slug: manifest
+order: 2
+layout: future
+---
+
+
+<div class="section-content">
+
+Your project should contain a `manifest.json` file that defines your app's manifest. This is where you'll configure your application name and scopes, declare the functions your app will use, and more.
+Refer to the <a href="https://api.slack.com/reference/manifests" target="_blank">App Manifest documentation</a> to learn
```

Comment:
Should we link to https://api.slack.com/reference/manifests, or maybe better to the next-gen page on manifest properties: https://api.slack.com/future/manifest#manifest-properties ?

Reply:
I'll add both 💯